ncm-spma/src/test/perl/apt-sources-generate.t

```diff
@@ -0,0 +1,108 @@
+# -*- mode: cperl -*-
+# ${license-info}
+# ${author-info}
+# ${build-info}
+
+=pod
+
+=head1 DESCRIPTION
+
+Tests for the generate_sources method.  They will ensure that entries in
+/etc/apt.lists.d are properly handled.
+
+=cut
+
+use strict;
+use warnings;
+use Test::Quattor;
+use Test::More;
+use NCM::Component::spma::apt;
+use Test::MockModule;
+use Readonly;
+use CAF::Object;
+use CAF::FileWriter;
+use Cwd;
+
+$CAF::Object::NoAction = 1;
```

Comment:
if you use `Test::Quattor`, this is only needed to test noaction behaviour, not to prevent that the unittests actually write stuff.